cat-gradle-sass-plugin/src/main/groovy/cc/catalysts/gradle/sass/task/Sass2Css.groovy

```diff
@@ -107,14 +83,9 @@ class Sass2Css extends NodeTask {
                     .findAll({ it.moduleVersion.id.group.startsWith('org.webjars') })
                     .forEach({ ResolvedArtifact it ->
                 String artifactId = "${it.name.replace('.', '-')}"
-                globalVars.put("webjars-${artifactId}", "webjars/${it.name}/${it.moduleVersion.id.version}")
             })
         });
 
-        for (Map.Entry<String, String> globalVar : globalVars) {
-            commonArgs.add("--global-var=${globalVar.key}='${globalVar.value}'")
```

Comment:
we still need support for importing our extracted webjar stylesheets in a simple way

so for example with bootstrap we need something like `@import "~webjars/bootstrap"`; or some other way which doesn't include the actual version -> the version has to be resolved by our config- maybe this can be accomplished via the cli (as with the less plugin) our you may need to wrap the actuall sass call